products/sle/main.pm

```diff
@@ -1142,7 +1142,15 @@ elsif (get_var("WICKED")) {
     load_wicked_tests();
 }
 elsif (get_var('NFV')) {
-    load_nfv_tests();
+    if (check_var("ROLE", "master")) {
```

Comment:
of course you can leave it like this , but what if you will just reuse NFV variable ? Also in case you will decide to keep it - please make name more specific - NFV_ROLE for example

Reply:
ok, load_nfv_tests is removed now